lib/internal/cluster/master.js

```diff
@@ -26,6 +26,7 @@ cluster.SCHED_RR = SCHED_RR;      // Master distributes connections.
 var ids = 0;
 var debugPortOffset = 1;
 var initialized = false;
+let debugSettingsOverriden = false;
```

Comment:
a global, augh...
&nbsp;💡 
🤓 
create a `Symbol` and hang it on `cluster.settings`
```js
const kDebugSettingsOverriden = Symbol('Debug Settings Overriden');
...
    var settings = {
      args: process.argv.slice(2),
      exec: process.argv[1],
      execArgv: process.execArgv,
      silent: false,
      [kDebugSettingsOverriden]: false
    };
```
What the heck, hang them all (maybe in the different PR though)